### DISCUSSION TITLE
Can you add create password protected zip?

### DISCUSSION THREAD

**Discussion by CodeLyokoXtEAM** [Studio-42/elFinder#3304] [Ideas]
Can you add Zip fie created with password protected?

&bug- I try to extract password protected zip file in elfinder but 0 byte in all files.

Found usefull site,
http://php.net/manual/en/class.ziparchive.php

**Comment by applibs** [comment]
ziparchive native class, doesnt support create password protected archive. Only unzip is possible.

**Comment by CodeLyokoXtEAM** [comment]
@standus 
This is only suggestion but I noticed someone try to unzip password protected zip in elfinder, its impossible

**Comment by CodeLyokoXtEAM** [comment]
But its impossible to rename folders inside zip
